lib/file.cpp

```diff
@@ -40,4 +51,21 @@ std::vector<utf8string> file$lines(const std::string &filename)
     return r;
 }
 
+void file$writelines(const std::string &filename, const std::vector<utf8string> &lines)
+{
+    std::ofstream f(filename, std::ios::out | std::ios::trunc);
+    if (not f.is_open()) {
+        throw RtlException("FileOpenError", filename);
+    }
+    for (auto s: lines) {
+        if (not s.empty()) {
```

Comment:
You know, this would be simplified if you remove the check for `s.empty()`, and do a `f << s.str() << "\n"` for each line. Writing no characters to a file (for blank lines) is not going to be a performance problem.